deploy/helm/kubecf/assets/operations/instance_groups/smoke-tests.yaml

```
@@ -4,9 +4,14 @@
   path: /instance_groups/name=smoke-tests/jobs/name=cf-cli-6-linux/properties?/quarks/bpm/processes
   value: []
 
-# The smoke-tests require the cf-cli package that is provided by the cf-cli-6-linux job. Since the
-# /var/vcap/packages directory is not shared between the containers, smoke-tests won't have access
-# to the cf-cli package. As part of the pre-render scripts added here will patch the cf-cli-6-linux
+- type: replace
+  path: /instance_groups/name=smoke-tests/env?/bosh?/agent?/settings?/disableLogSidecar?
```

You don't need `?` on every part of the path.

```suggestion
  path: /instance_groups/name=smoke-tests/env?/bosh/agent/settings/disableLogSidecar
```